src/index.js

```diff
@@ -77,6 +78,14 @@ class Client {
     let unsupported = [];
 
     if (version) {
+      // Convert to semver (removing pathes).
+      if (!/[0-9]+\.[0-9]+\.[0-9]+/.test(version)) {
```

Comment:
I don't think we need to run this test as exec gives us both informations (the test and the capture which we need below).